content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -251,6 +249,9 @@ For a reference to old feature gates that are removed, please refer to
 | `DefaultPodTopologySpread` | `false` | Alpha | 1.19 | 1.19 |
 | `DefaultPodTopologySpread` | `true` | Beta | 1.20 | 1.23 |
 | `DefaultPodTopologySpread` | `true` | GA | 1.24 | - |
+| `DevicePlugins` | `false` | Alpha | 1.8 | 1.9 |
+| `DevicePlugins` | `true` | Beta | 1.10 | |
```

Comment:
asking, not sure myself: should this read like
```
`DevicePlugins` | `true` | Beta | 1.10 | 1.25 |
```
?

Reply:
Additionally, since we're graduating to GA, do we need to keep this content in this file or should we move this into the `feature-gates-removed.md` file? Again asking, I'm not sure myself about the process.

Reply:
Going to go ahead and add the following (as I think it should have been added initially itself):
```
DevicePlugins` | `true` | Beta | 1.10 | 1.25 |
```
Will wait for input on whether this needs to go to into the `feature-gates-removed.md` file or not.

Reply:
Found this text https://github.com/kubernetes/website/blame/main/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md#L9-L18 that indicates that we shouldn't have feature gates corresponding to GA features in `feature-gates-removed.md`.